Source/driver/Castro.cpp

```diff
@@ -1037,6 +1037,11 @@ Castro::initData ()
 #endif
 #endif
 
+#ifdef NSE_NET
+   MultiFab& ChemPot_new = get_new_data(Chemical_Pot_Type);
+   ChemPot_new.setVal(0.);
+#endif
```

Comment:
you should add `ChemPot_new.nGrow()` to the `setVal` so the ghost cells are also initialized to 0